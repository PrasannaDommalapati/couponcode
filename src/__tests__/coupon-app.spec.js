import React from 'react';
import CouponApp from '../components/coupon-app';
import renderer from 'react-test-renderer';

describe('App Component', () => {
  it('renders App coponent correctly', () => {
    const tree = renderer.create(<CouponApp/>).toJSON();
    // expect(tree).toMatchSnapshot();
  });
});
