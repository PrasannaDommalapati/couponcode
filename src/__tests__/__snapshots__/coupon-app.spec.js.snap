// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component renders App coponent correctly 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <h1>
      Welcome to Coupon code Appppp
    </h1>
  </header>
</div>
`;
